versioned_docs/version-23.10/health/centreon-agent.md

```diff
@@ -393,9 +393,9 @@ systemctl restart centreon-agent.service
 
 ### Tags
 
-The Agent can contextualize data collection with your own custom tags to define the perimeter in which it is in action. This is used later on to aggregate the monitoring data around your tags and create dashboards or reports in relevant contexts.
+The Agent can contextualize data collection with your own custom tags to define the perimeter in which it operates. This is used later on to aggregate the monitoring data around your tags and create dashboards or reports in relevant contexts.
 
->We strongly advise the first tag you define to be “environment” in order for us to be able to establish a common baseline between all users.
+>We strongly advise you to define the “environment” tag before any others, so that we can establish a common baseline between all users.
```

Comment:
```suggestion
> We strongly advise you to define the “environment” tag before any others, so that we can establish a common baseline between all users.
```